src/api/app/views/webui/status_messages/acknowledge.js.haml

```diff
@@ -0,0 +1,5 @@
+- if flash[:error]
```

Comment:
You are not using haml inside this partial, this is ruby. Should be `acknowledge.js.erb`

Reply:
IMO, they are using HAML. or I'm wrong?

Reply:
We are using haml. We could use `<% if ` instead of `- if`, does it have any benefit?

Reply:
ups. did not see this. sorry